app/controllers/concerns/query_sanitizer.rb

```diff
@@ -0,0 +1,5 @@
+module QuerySanitizer
+  def strip_non_ascii(query)
```

Comment:
Naming stuff is hard. Let me know if you have better alternatives

Reply:
is it actually striping it or encoding it :thinking:

Reply:
Stripping